examples/platform/silabs/provision/ProvisionStorageFlash.cpp

```diff
@@ -664,7 +664,7 @@ CHIP_ERROR Storage::SignWithDeviceAttestationKey(const ByteSpan & message, Mutab
     }
 #endif // SL_MATTER_ENABLE_EXAMPLE_CREDENTIALS
     ReturnErrorOnFailure(err);
-#if (defined(SLI_SI91X_MCU_INTERFACE) && SLI_SI91X_MCU_INTERFACE)
+#if (defined(SL_MBEDTLS_USE_TINYCRYPT) && SL_MBEDTLS_USE_TINYCRYPT)
```

Comment:
```suggestion
#ifdef SL_MBEDTLS_USE_TINYCRYPT
```

Why not just that?
The codebase is full of usages like the one above. Might as well make it consistent